README.md

```diff
@@ -1 +1,23 @@
-# Setup-and-mobile-version-skeleton
\ No newline at end of file
+# Setup-and-mobile-version-skeleton
+
+> This is my portfolio Microverse project
+Additional description about the project and its features.
+
+![screenshot](./Portfolio-screenshot.png)
+## Built With
+
+- html
+- css
+- github
+
```

Comment:
**OPTIONAL:** Please try to give descriptive names on feature branches, like 'created_header' or 'added_comments_section' in your future projects, so it will show you what you have done for this feature.

_(I set it OPTIONAL because if you change your branch name now, your PR will be closed, but please do not repeat this mistake for other projects)_